packages/scripts/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.0 (2018-05-23)
```

Comment:
I think we will wait for license checker before we release `1.2.0`.

Reply:
>I think we will wait for license checker before we release `1.2.0`.

 So will this or won't this be published on `2018-05-23` ? I think we should either:

- Not compile `CHANGELOG.md` edits until immediately before a published release.
   - This is what I thought had been settled upon in a previous core-js meeting ([reference](https://wordpress.slack.com/archives/C5UNMSU4R/p1524577254000794))
- Not merge with an explicit date for a version until the date is known (either omit entirely, or a placeholder value like "Unreleased" or "TBD").

Regardless, we should codify this in `CONTRIBUTING.md` workflow documentation.

Reply:
Yes, `Unreleased` seems like a good choice in this case. I can publish new version just to make sure it doesn't lie 😄

Reply:
For the record, it is still unpublished, we should update docs and follow the flow @aduth described 👍